src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private static final int MAX_VALUE = 101;
```

Comment:
The value of MAX_VALUE should be 100, not 101. The task specifies that the maximum number for a ball should be 100. You can add 1 to this value in random number generation, but this constant should be 100.

Reply:
done